docs-next/components/Navigation.tsx

```diff
@@ -64,6 +64,7 @@ export const Navigation = () => {
         <NavItem href="/guides/testing" isPlaceholder>
           Testing
         </NavItem>
+        <NavItem href="/guides/keystone-5-vs-keystone-next">Keystone 5 vs Next</NavItem>
```

Comment:
Can we move this to the top of the guides block? I think this is probably the most important guide that we have at the moment (or possibly number 2, after the command line).